project/core/settings.py

```diff
@@ -6,8 +6,8 @@
 """
 import os
 
-# False if not in os.environ
-DEBUG = os.getenv("DEBUG", False)
+# True if not in os.environ
+DEBUG = os.getenv("DEBUG", True)
```

Comment:
For security, we want the default value for `DEBUG` to be `False`
```suggestion
# True if not in os.environ
DEBUG = os.getenv("DEBUG", False)
```

Reply:
kk i'll leave it as default and push up the changes